### ISSUE TITLE
How to call custom endpoint?

### ISSUE THREAD

**Issue by smyth64** [feathers-plus/feathers-vuex#219]
Hi,
I have a custom endpoint that is not the default CRUD...

Like /team/invite-external-member or something.

How can I call this using the plugin? Or do I have to use axios here?

**Comment by marshallswain** [comment]
Feathers is all-in on REST (the default CRUD), so the key is to either map your resources into REST (I've never found a scenario that didn't fit with slight variation) OR do the DIY version with Axios.

Your example is troublesome for two reasons.  First, because it focuses on the verb, but there is a noun in plain sight.  Just focus on the noun.  

- `/team/invite-external-member` <-- Verb version
- `/team/invites` <-- Noun version

Second: Notice that I've also removed the "external-member" wording, because that's *data*.  That `external` status will almost certainly end up as `isExternal: true` on some record (or some form of that).   Your API will become much easier to maintain in the long run if you keep the data in the payload.

There's a potential third pitfall to your URL scheme depending on how you're using the routes.  Nested routes don't do anybody a favor, so technically the URL can become even simpler.

- `/team-invites`
- `/invites`

with payload `{ teamId: 5 }`.  The way you're using it, currently, it's not REALLY an issue.  As soon as you try to do URL params, like `/teams/:teamId/invites` you're providing a user experience where none is really necessary.  It always ends up in being extra work for almost no reason.  I've only seen one scenario where it was really "required" to do nested route params.  The company had a proxy server that would ready the route params and would direct the traffic to the correct server based on those params.  This wasn't done for the sake of pretty URLs, it was all because the proxy server could redirect the request very quickly.

Hopefully I've answered your question somewhere in all of my unsolicited advice.  ;)
